tracks/bash/exercises/hamming/mentoring.md

```diff
@@ -0,0 +1,147 @@
+### Reasonable Solutions
+
+```bash
+#!/usr/bin/env bash
+main () {
+  (( "$#" == 2 )) || {
```

Comment:
Using an `if` conditional makes things a whole lot easier to read and less prone to pitfalls. The `if` conditional ought to be recommended over a multi-line `&&`